python-frontend/src/main/java/org/sonar/plugins/python/api/tree/IfStatement.java

```diff
@@ -23,7 +23,19 @@
 import javax.annotation.CheckForNull;
 
 /**
- * if-elif-else statement
+ * if-elif-else statement.
+ *
+ * Note that this interface has a recursive structure because it represents both 'if' clause and 'elif' clause
+ *
+ * <pre>
+ *   if {@link #condition()}:
+ *     {@link #body()}
+ *   elif {@link #condition()}:
+ *     {@link #body()}
+ *   elif {@link #condition()}:
+ *     {@link #body()}
+ *   {@link #elseBranch()}
+ * </pre>
```

Comment:
Having `{@link #condition()}` and `{@link #body()}` appearing multiple times here seems misleading to me.
In fact, maybe I don't like our AST model for `elif` clauses.